[Defend Workflows][Trusted Apps] Advanced mode toggle and form

parkiino opened the pull request:
## Summary

- [x] UI portion for advanced mode form in Trusted Apps page adds a toggle for the user to switch between the original basic conditions mode or the advanced mode. 
- [x] If the user toggles between forms after building some conditions, the state is saved per form while the flyout is open.
- [x] When editing an existing trusted app, the saved form type is pre-populated in the conditions section
- [x] Advanced mode contains additional text to warn the user of the implications of using advanced mode for trusted apps 
- [x] Updates search strategy and suggestions api to include trusted apps
- [x] Unit tests

# Screenshots

<img width="1641" alt="image" src="https://github.com/user-attachments/assets/24edb4ef-aa86-4dd5-86b7-c37993afe512" />

Commit:
api changes for suggestions and search strat

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
experimental flag and advanced mode button

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
working advanced mode toggle that updates tag

Commit:
update ta api client and add search strat and suggestions

Commit:
save advanced and basic entries in state and update when toggle changes

Commit:
wip add exception builder component

Commit:
apply konrads patch + fix suggestions request type

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
adjust spacing, remove unused logger prop, move warning placement

Commit:
do not do write/read transform for advanced mode entries

Commit:
fix mode change and add advanced mode warning

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/styled_components_mapping'

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/316493)
* Commit: 9da7d6dc68fef9ade6e869fd21560425e69682ef

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/316493#0197e62f-eb15-483f-8ad4-08c5d5244a0a)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316493","number":316493,"commit":"9da7d6dc68fef9ade6e869fd21560425e69682ef"}],"number":316493}
buildkite-pr-comment-->

Commit:
fix logic to show warning text and advanced mode form

Commit:
Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

Commit:
linting

Commit:
move common utils, adjust button group to be right aligned

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/316580)
* Commit: e086f88e2bd99602ab272b8131042417b2466a6f

### Failed CI Steps
* [Jest Tests #4](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771)
* [Jest Tests #10](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9d-49eb-8d51-591413c5f218)
* [Jest Tests #10](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8dd-df43-4d05-b7f8-66ed3bbe7fca)
* [FTR Configs #106](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-baf0-40b4-a10e-e7ad14b49213)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec45-4ff4-b4b4-5eb9cf6a90c0) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder and when the AND button is clicked should add a new condition entry when `AND` is clicked with no column labels
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec8a-4620-9a64-0925b47a20bd) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder and when the AND button is clicked should have remove buttons enabled when multiple conditions are present
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec8a-4dc8-b06a-865faa639240) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder IS operator should show path malformed path warning for linux/mac without an executable name
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec89-4bfe-aa10-1c13f0402338) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder IS operator should show path malformed path warning for windows with no executable name
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec89-40d9-9c1c-1b2f1275698b) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder IS operator should show path malformed warning
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec48-4e91-abaa-18bddc4f69f9) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder MATCHES operator should show wildcard in path warning
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec45-4d96-a664-86e4f0d23739) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder should default to hash entry field
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec87-4c94-b2ec-38d4ff4fb4ac) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder should display 3 options for Field for Windows
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec89-407c-8447-010920a706d9) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder should display the `AND` button
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec88-4cee-b57f-847fa7d7c325) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder should not allow the entry to be removed if its the only one displayed
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec89-4920-a1c5-0e32912f7b0b) Jest Tests #<span></span>4 / Trusted apps form ConditionBuilder should show the value field as required after blur
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec87-462d-be58-04a87e4e1362) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should allow user to select between 3 OSs
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec88-4be2-96da-4617de3a8e99) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should be invalid if no name
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec87-457f-aaf9-1de9d010160a) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should correctly change OS
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec87-4f32-89d6-b7854bdde293) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should correctly edit description
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec44-44ae-9d48-1f63cab77949) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should correctly edit name
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec88-461a-8601-f7d3e281c183) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should default OS to Windows
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec45-4d71-b5b2-32e338a7d3fa) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should show Description as optional
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec47-4a2d-a68b-7142880721f2) Jest Tests #<span></span>4 / Trusted apps form Details and Conditions should show name required name blur
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9c-403f-80ff-11badc116771) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9c-403f-80ff-11badc116771/artifacts/0197e8dc-ec89-4982-9565-a863a0e548c0) Jest Tests #<span></span>4 / Trusted apps form should display form submission errors
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8ba-ba9d-49eb-8d51-591413c5f218) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8ba-ba9d-49eb-8d51-591413c5f218/artifacts/0197e8dd-94a5-4710-a2d7-385303a3cc19) Jest Tests #<span></span>10 / when calling the Suggestions route handler having right privileges without having right privileges should respond with forbidden
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316580#0197e8dd-df43-4d05-b7f8-66ed3bbe7fca) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316580/jobs/0197e8dd-df43-4d05-b7f8-66ed3bbe7fca/artifacts/0197e8ea-c51a-40d7-bedf-42ef5252a658) Jest Tests #<span></span>10 / when calling the Suggestions route handler having right privileges without having right privileges should respond with forbidden

### History
* :broken_heart: [Build #316493](https://buildkite.com/elastic/kibana-pull-request/builds/316493) failed 9da7d6dc68fef9ade6e869fd21560425e69682ef

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316580","number":316580,"commit":"e086f88e2bd99602ab272b8131042417b2466a6f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316493","number":316493,"commit":"9da7d6dc68fef9ade6e869fd21560425e69682ef"}],"number":316580}
buildkite-pr-comment-->

Commit:
fix broken test

Commit:
Mt push:wq::

Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/317073)
* Commit: 8d1355301e0668b01f32f5c3c051b74bac8d18c2

### Failed CI Steps
* [Jest Tests #13](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f1-4e4a-9a6e-a58cef18effc)
* [Jest Tests #15](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f2-43a4-9eed-78c248ca5d4d)
* [Jest Tests #15](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edef-b15a-4ca2-9229-18c745462421)
* [Jest Integration Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f4-4093-8fc3-55d5dbbbb9d4)
* [FTR Configs #8](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-4712-448f-b857-94388e5c3626)
* [FTR Configs #13](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-4718-4a0b-b106-88c7ee965463)
* [FTR Configs #18](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-471c-4d5d-b1d5-3d5647539a50)
* [FTR Configs #35](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edd4-9693-48a0-a3e4-d2287989a9a3)
* [FTR Configs #40](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-472f-4734-ae9a-2eb5e1f105e3)
* [FTR Configs #59](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-4748-4464-afb7-16c8f3bdb903)
* [FTR Configs #80](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-4758-462a-bad7-5cd291607d70)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f1-4e4a-9a6e-a58cef18effc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f1-4e4a-9a6e-a58cef18effc/artifacts/0197edeb-5271-47d1-97b2-9b15a3a8257a) Jest Tests #<span></span>13 / Condition entry input should be able to select three options when MAC OS
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f1-4e4a-9a6e-a58cef18effc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f1-4e4a-9a6e-a58cef18effc/artifacts/0197edeb-5270-4dad-b7f7-74b8b5470d16) Jest Tests #<span></span>13 / Condition entry input should be able to select three options when WINDOWS OS
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f1-4e4a-9a6e-a58cef18effc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f1-4e4a-9a6e-a58cef18effc/artifacts/0197edeb-527b-4166-943b-8af1b96ebf07) Jest Tests #<span></span>13 / Condition entry input should be able to select two options when LINUX OS
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f1-4e4a-9a6e-a58cef18effc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f1-4e4a-9a6e-a58cef18effc/artifacts/0197edeb-5270-4e43-b644-4aff24f4a2e3) Jest Tests #<span></span>13 / Condition entry input should call on remove for field input
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f1-4e4a-9a6e-a58cef18effc) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f1-4e4a-9a6e-a58cef18effc/artifacts/0197edeb-5270-4c0a-a4bb-14291f98aece) Jest Tests #<span></span>13 / Condition entry input should not be able to call on remove for field input because disabled
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f4-4093-8fc3-55d5dbbbb9d4) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f4-4093-8fc3-55d5dbbbb9d4/artifacts/0197eded-3ac1-4aea-b7ba-8d7ae839b80a) Jest Integration Tests #<span></span>1 / Fleet preconfiguration reset Reset all policy Works and reset all preconfigured policies
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f4-4093-8fc3-55d5dbbbb9d4) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f4-4093-8fc3-55d5dbbbb9d4/artifacts/0197eded-3ac1-4457-8589-5928213eec55) Jest Integration Tests #<span></span>1 / Fleet preconfiguration reset Reset one preconfigured policy Works and reset one preconfigured policies if the policy is already deleted (with a ghost package policy)
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f4-4093-8fc3-55d5dbbbb9d4) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f4-4093-8fc3-55d5dbbbb9d4/artifacts/0197eded-3ac1-4a8a-bcd9-f6c2cc50fc4b) Jest Integration Tests #<span></span>1 / Fleet preconfiguration reset Reset one preconfigured policy Works and reset one preconfigured policies if the policy was deleted with a preconfiguration deletion record
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f4-4093-8fc3-55d5dbbbb9d4) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f4-4093-8fc3-55d5dbbbb9d4/artifacts/0197eded-3ac0-40ad-a529-52caa01ecec7) Jest Integration Tests #<span></span>1 / Fleet preconfiguration reset Reset one preconfigured policy Works if the preconfigured policies already exists with a missing package policy
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edc7-46f2-43a4-9eed-78c248ca5d4d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edc7-46f2-43a4-9eed-78c248ca5d4d/artifacts/0197edef-4e7a-4c27-bb2e-a014c091edb0) Jest Tests #<span></span>15 / when calling the Suggestions route handler having right privileges without having right privileges should respond with forbidden
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/317073#0197edef-b15a-4ca2-9229-18c745462421) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/317073/jobs/0197edef-b15a-4ca2-9229-18c745462421/artifacts/0197edf9-73d2-461b-9a7b-0a65ad5940a1) Jest Tests #<span></span>15 / when calling the Suggestions route handler having right privileges without having right privileges should respond with forbidden

### History
* :broken_heart: [Build #316580](https://buildkite.com/elastic/kibana-pull-request/builds/316580) failed e086f88e2bd99602ab272b8131042417b2466a6f
* :broken_heart: [Build #316493](https://buildkite.com/elastic/kibana-pull-request/builds/316493) failed 9da7d6dc68fef9ade6e869fd21560425e69682ef

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317073","number":317073,"commit":"8d1355301e0668b01f32f5c3c051b74bac8d18c2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316580","number":316580,"commit":"e086f88e2bd99602ab272b8131042417b2466a6f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316493","number":316493,"commit":"9da7d6dc68fef9ade6e869fd21560425e69682ef"}],"number":317073}
buildkite-pr-comment-->

Commit:
trusted apps operators not modified, renamed event filters operators list to generic

Commit:
Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

gergoabraham reviewed (commented):
tested it, works well! 👏

tomsonpl committed:
Merge branch 'main' into task/api-TA-endpointSearchStrat-suggestions

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

tomsonpl reviewed (approved):
Thanks @parkiino - this looks good 👍 I thing everything works as expected (tried my best to test it out). 👏 I didn't see anything blocking this PR, other than what @gergoabraham already mentioned :) 

However, I see that each of the PR's added to these forms are adding to it's complexity, and I think might be the time soon to start refactoring it, the validation logic, the rendering logic look overly complex with all the if else statements. So maybe before taking on a new task, talk with @dasansol92 about having some time to rethink the approach. I am sure it will help you in the future when new features are to be implemented.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/318662)
* Commit: a65b0eb068d63f1b741764b4899ecdea3ae4c1b5

### Failed CI Steps
* [Linting](https://buildkite.com/elastic/kibana-pull-request/builds/318662#01980873-3978-4506-a8b9-c06430892bac)

### History
* :broken_heart: [Build #318003](https://buildkite.com/elastic/kibana-pull-request/builds/318003) failed 0597e5b45ae4de7b05717d337977efccf3692952
* :broken_heart: [Build #317073](https://buildkite.com/elastic/kibana-pull-request/builds/317073) failed 8d1355301e0668b01f32f5c3c051b74bac8d18c2
* :broken_heart: [Build #316580](https://buildkite.com/elastic/kibana-pull-request/builds/316580) failed e086f88e2bd99602ab272b8131042417b2466a6f
* :broken_heart: [Build #316493](https://buildkite.com/elastic/kibana-pull-request/builds/316493) failed 9da7d6dc68fef9ade6e869fd21560425e69682ef

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318662","number":318662,"commit":"a65b0eb068d63f1b741764b4899ecdea3ae4c1b5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318003","number":318003,"commit":"0597e5b45ae4de7b05717d337977efccf3692952"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317073","number":317073,"commit":"8d1355301e0668b01f32f5c3c051b74bac8d18c2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316580","number":316580,"commit":"e086f88e2bd99602ab272b8131042417b2466a6f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316493","number":316493,"commit":"9da7d6dc68fef9ade6e869fd21560425e69682ef"}],"number":318662}
buildkite-pr-comment-->

Commit:
fix broken tests

Commit:
Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/lint_ts_projects --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/320672)
* Commit: b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/320672#01981bf2-5695-4748-b2d5-06efabd8378d)

### History
* :broken_heart: [Build #318662](https://buildkite.com/elastic/kibana-pull-request/builds/318662) failed a65b0eb068d63f1b741764b4899ecdea3ae4c1b5
* :broken_heart: [Build #318003](https://buildkite.com/elastic/kibana-pull-request/builds/318003) failed 0597e5b45ae4de7b05717d337977efccf3692952
* :broken_heart: [Build #317073](https://buildkite.com/elastic/kibana-pull-request/builds/317073) failed 8d1355301e0668b01f32f5c3c051b74bac8d18c2
* :broken_heart: [Build #316580](https://buildkite.com/elastic/kibana-pull-request/builds/316580) failed e086f88e2bd99602ab272b8131042417b2466a6f
* :broken_heart: [Build #316493](https://buildkite.com/elastic/kibana-pull-request/builds/316493) failed 9da7d6dc68fef9ade6e869fd21560425e69682ef

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320672","number":320672,"commit":"b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318662","number":318662,"commit":"a65b0eb068d63f1b741764b4899ecdea3ae4c1b5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318003","number":318003,"commit":"0597e5b45ae4de7b05717d337977efccf3692952"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317073","number":317073,"commit":"8d1355301e0668b01f32f5c3c051b74bac8d18c2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316580","number":316580,"commit":"e086f88e2bd99602ab272b8131042417b2466a6f"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316493","number":316493,"commit":"9da7d6dc68fef9ade6e869fd21560425e69682ef"}],"number":320672}
buildkite-pr-comment-->

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/320673)
* Commit: ed1232f5fcd084e288ffe31ec384e905eb9e30f8

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/320673#01981c09-9eb6-4934-8268-a7e8e2acf804)

### History
* :broken_heart: [Build #320672](https://buildkite.com/elastic/kibana-pull-request/builds/320672) failed b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7
* :broken_heart: [Build #318662](https://buildkite.com/elastic/kibana-pull-request/builds/318662) failed a65b0eb068d63f1b741764b4899ecdea3ae4c1b5
* :broken_heart: [Build #318003](https://buildkite.com/elastic/kibana-pull-request/builds/318003) failed 0597e5b45ae4de7b05717d337977efccf3692952
* :broken_heart: [Build #317073](https://buildkite.com/elastic/kibana-pull-request/builds/317073) failed 8d1355301e0668b01f32f5c3c051b74bac8d18c2
* :broken_heart: [Build #316580](https://buildkite.com/elastic/kibana-pull-request/builds/316580) failed e086f88e2bd99602ab272b8131042417b2466a6f
* :broken_heart: [Build #316493](https://buildkite.com/elastic/kibana-pull-request/builds/316493) failed 9da7d6dc68fef9ade6e869fd21560425e69682ef

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320673","number":320673,"commit":"ed1232f5fcd084e288ffe31ec384e905eb9e30f8"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320672","number":320672,"commit":"b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318662","number":318662,"commit":"a65b0eb068d63f1b741764b4899ecdea3ae4c1b5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318003","number":318003,"commit":"0597e5b45ae4de7b05717d337977efccf3692952"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317073","number":317073,"commit":"8d1355301e0668b01f32f5c3c051b74bac8d18c2"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316580","number":316580,"commit":"e086f88e2bd99602ab272b8131042417b2466a6f"}],"number":320673}
buildkite-pr-comment-->

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/lint_ts_projects --fix'

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/320674)
* Commit: 53b8c96b5d756127d2fdb9f94e5abba3a1d478ca

### Failed CI Steps
* [Defend Workflows Cypress Tests #3](https://buildkite.com/elastic/kibana-pull-request/builds/320674#01981c19-77d2-4bfd-8de9-bdee6443acf9)

### History
* :broken_heart: [Build #320673](https://buildkite.com/elastic/kibana-pull-request/builds/320673) failed ed1232f5fcd084e288ffe31ec384e905eb9e30f8
* :broken_heart: [Build #320672](https://buildkite.com/elastic/kibana-pull-request/builds/320672) failed b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7
* :broken_heart: [Build #318662](https://buildkite.com/elastic/kibana-pull-request/builds/318662) failed a65b0eb068d63f1b741764b4899ecdea3ae4c1b5
* :broken_heart: [Build #318003](https://buildkite.com/elastic/kibana-pull-request/builds/318003) failed 0597e5b45ae4de7b05717d337977efccf3692952
* :broken_heart: [Build #317073](https://buildkite.com/elastic/kibana-pull-request/builds/317073) failed 8d1355301e0668b01f32f5c3c051b74bac8d18c2
* :broken_heart: [Build #316580](https://buildkite.com/elastic/kibana-pull-request/builds/316580) failed e086f88e2bd99602ab272b8131042417b2466a6f

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320674","number":320674,"commit":"53b8c96b5d756127d2fdb9f94e5abba3a1d478ca"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320673","number":320673,"commit":"ed1232f5fcd084e288ffe31ec384e905eb9e30f8"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320672","number":320672,"commit":"b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318662","number":318662,"commit":"a65b0eb068d63f1b741764b4899ecdea3ae4c1b5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318003","number":318003,"commit":"0597e5b45ae4de7b05717d337977efccf3692952"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317073","number":317073,"commit":"8d1355301e0668b01f32f5c3c051b74bac8d18c2"}],"number":320674}
buildkite-pr-comment-->

Commit:
text chagnes

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
wip form tests

Commit:
Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

Commit:
tests for now

Commit:
Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/323145)
* Commit: 558feea0f74c7de34ff3a68365f062ef71bbcf78

### Failed CI Steps
* [Osquery Cypress Tests on Serverless #2](https://buildkite.com/elastic/kibana-pull-request/builds/323145#01984056-8034-4a6a-b858-c5117eafd2b5)

### History
* :broken_heart: [Build #322657](https://buildkite.com/elastic/kibana-pull-request/builds/322657) failed 212af77bbc5286d68188730e0cada21e0ecd0cbb
* :broken_heart: [Build #320674](https://buildkite.com/elastic/kibana-pull-request/builds/320674) failed 53b8c96b5d756127d2fdb9f94e5abba3a1d478ca
* :broken_heart: [Build #320673](https://buildkite.com/elastic/kibana-pull-request/builds/320673) failed ed1232f5fcd084e288ffe31ec384e905eb9e30f8
* :broken_heart: [Build #320672](https://buildkite.com/elastic/kibana-pull-request/builds/320672) failed b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7
* :broken_heart: [Build #318662](https://buildkite.com/elastic/kibana-pull-request/builds/318662) failed a65b0eb068d63f1b741764b4899ecdea3ae4c1b5

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/323145","number":323145,"commit":"558feea0f74c7de34ff3a68365f062ef71bbcf78"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322657","number":322657,"commit":"212af77bbc5286d68188730e0cada21e0ecd0cbb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320674","number":320674,"commit":"53b8c96b5d756127d2fdb9f94e5abba3a1d478ca"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320673","number":320673,"commit":"ed1232f5fcd084e288ffe31ec384e905eb9e30f8"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320672","number":320672,"commit":"b5100cad24f4254294f6f6a6cc3f1e1b13ed1de7"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/318662","number":318662,"commit":"a65b0eb068d63f1b741764b4899ecdea3ae4c1b5"}],"number":323145}
buildkite-pr-comment-->

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
fix tests add feature flag mock

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/324094)
* Commit: 1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b

### Failed CI Steps
* [FTR Configs #107](https://buildkite.com/elastic/kibana-pull-request/builds/324094#019852f1-31f1-489b-a4dd-31f215bff6c3)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/324094#019852f1-31f1-489b-a4dd-31f215bff6c3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/324094/jobs/019852f1-31f1-489b-a4dd-31f215bff6c3/artifacts/01985304-d643-4078-bebe-eab594ff28a4) FTR Configs #<span></span>107 / serverless observability UI discover/observabilitySolution/context_awareness extension getDocViewer data view mode should not render logs overview tab for non-logs data source

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/c1dc137429bae0e62e16d187a431fde35d091a7d) | [after](https://github.com/elastic/kibana/commit/1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b) | diff |
| --- | --- | --- | --- |
| `@kbn/securitysolution-list-utils` | 162 | 161 | -1 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/c1dc137429bae0e62e16d187a431fde35d091a7d) | [after](https://github.com/elastic/kibana/commit/1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b) | diff |
| --- | --- | --- | --- |
| `lists` | 124.8KB | 125.3KB | +502.0B |
| `securitySolution` | 9.8MB | 9.8MB | +7.3KB |
| total |  |  | +7.8KB |

#### Count of Enzyme imports
> Enzyme is no longer supported, and we should switch to @testing-library/react instead.


| id | [before](https://github.com/elastic/kibana/commit/c1dc137429bae0e62e16d187a431fde35d091a7d) | [after](https://github.com/elastic/kibana/commit/1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 190 | 189 | -1 |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/c1dc137429bae0e62e16d187a431fde35d091a7d) | [after](https://github.com/elastic/kibana/commit/1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 94.3KB | 94.4KB | +30.0B |

<details><summary>Unknown metric groups</summary>

#### async chunk count
| id | [before](https://github.com/elastic/kibana/commit/c1dc137429bae0e62e16d187a431fde35d091a7d) | [after](https://github.com/elastic/kibana/commit/1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b) | diff |
| --- | --- | --- | --- |
| `securitySolution` | 101 | 102 | +1 |


</details>


### History
* :broken_heart: [Build #323521](https://buildkite.com/elastic/kibana-pull-request/builds/323521) failed 8b8da62d4d0cdeadbc0eeb95f76106b3fe8fc694
* :broken_heart: [Build #323145](https://buildkite.com/elastic/kibana-pull-request/builds/323145) failed 558feea0f74c7de34ff3a68365f062ef71bbcf78
* :broken_heart: [Build #322657](https://buildkite.com/elastic/kibana-pull-request/builds/322657) failed 212af77bbc5286d68188730e0cada21e0ecd0cbb
* :broken_heart: [Build #320674](https://buildkite.com/elastic/kibana-pull-request/builds/320674) failed 53b8c96b5d756127d2fdb9f94e5abba3a1d478ca
* :broken_heart: [Build #320673](https://buildkite.com/elastic/kibana-pull-request/builds/320673) failed ed1232f5fcd084e288ffe31ec384e905eb9e30f8

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/324094","number":324094,"commit":"1c5b6299fa82c42e8c08f8edc3d10eea5d24a40b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/323521","number":323521,"commit":"8b8da62d4d0cdeadbc0eeb95f76106b3fe8fc694"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/323145","number":323145,"commit":"558feea0f74c7de34ff3a68365f062ef71bbcf78"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/322657","number":322657,"commit":"212af77bbc5286d68188730e0cada21e0ecd0cbb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320674","number":320674,"commit":"53b8c96b5d756127d2fdb9f94e5abba3a1d478ca"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/320673","number":320673,"commit":"ed1232f5fcd084e288ffe31ec384e905eb9e30f8"}],"number":324094}
buildkite-pr-comment-->

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

Commit:
Merge branch 'task/api-TA-endpointSearchStrat-suggestions' of github.com:parkiino/kibana into task/api-TA-endpointSearchStrat-suggestions

Commit:
rename var

Commit:
Merge remote-tracking branch 'upstream/main' into task/api-TA-endpointSearchStrat-suggestions

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint_all_files --no-cache --fix'
